tests/chainer_tests/functions_tests/math_tests/test_exponential_m1.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(), (3, 2)],
+}))
+class Expm1FunctionTest(unittest.TestCase):
+
+    def setUp(self):
+        self.x = numpy.random.uniform(.5, 1, self.shape).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, self.shape).astype(numpy.float32)
+
+    def check_forward(self, op, op_np, x_data):
```

Comment:
Thank you for your comments.
I thought this `_np` means `Numpy` and this test tries to figure out that the results of the Chainer's `expm1` is almost same as Numpy's one.
So even if we test only `expm1`, this test seems to require both `op` and `op_np`.

Thanks.